curvefs/src/tools/BUILD

```diff
@@ -66,6 +67,7 @@ cc_library(
         "buildtopo/curvefs_build_topology_tool.h",
         "usage/curvefs_metadata_usage_tool.h",
         "usage/curvefs_space_base_tool.h",
+        "status/curvefs_status.h",
```

Comment:
you didn't get the idea of `glob`

```
glob([
   "*.h",
   "dir1/*.h",
   "dir2/*.h",
])
```

is enough

Reply:
fix